### ISSUE TITLE
zoneinfo uses locale depending functions for parsing

### ISSUE THREAD

**Issue by serhiy-storchaka** [python/cpython#86842]
BPO | [42676](https://bugs.python.org/issue42676)
--- | :---
Nosy | @serhiy-storchaka, @pganssle

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = None
closed_at = None
created_at = <Date 2020-12-18.18:09:32.183>
labels = ['type-bug', 'library', '3.9', '3.10']
title = 'zoneinfo uses locale depending functions for parsing'
updated_at = <Date 2020-12-18.18:09:32.183>
user = 'https://github.com/serhiy-storchaka'
```

bugs.python.org fields:
```python
activity = <Date 2020-12-18.18:09:32.183>
actor = 'serhiy.storchaka'
assignee = 'none'
closed = False
closed_date = None
closer = None
components = ['Library (Lib)']
creation = <Date 2020-12-18.18:09:32.183>
creator = 'serhiy.storchaka'
dependencies = []
files = []
hgrepos = []
issue_num = 42676
keywords = []
message_count = 1.0
messages = ['383313']
nosy_count = 2.0
nosy_names = ['serhiy.storchaka', 'p-ganssle']
pr_nums = []
priority = 'normal'
resolution = None
stage = None
status = 'open'
superseder = None
type = 'behavior'
url = 'https://bugs.python.org/issue42676'
versions = ['Python 3.9', 'Python 3.10']
```

</p></details>

**Comment by serhiy-storchaka** [comment]
Fixed in #108765.

**Comment by vstinner** [comment]
> Fixed in https://github.com/python/cpython/issues/108767.

Oh nice, I didn't know this issue.
